lib/active_model/serializer/associations.rb

```diff
@@ -13,9 +13,10 @@ module Associations
       DEFAULT_INCLUDE_TREE = ActiveModel::Serializer::IncludeTree.from_string('*')
 
       included do |base|
-        class << base
-          attr_accessor :_reflections
-        end
+        base.class_attribute :serialized_associations, instance_writer: false # @api public: maps association name to 'Reflection' instance
```

Comment:
should probably follow the same convention in the Serializer and be `_association_mappings`